pkg/apis/core/types.go

```diff
@@ -2597,8 +2597,18 @@ type PodSpec struct {
 	// This is an alpha feature and may change in the future.
 	// +optional
 	RuntimeClassName *string
+	// EnableServiceLinks indicates whether information about services should be injected into pod's
+	// environment variables, matching the syntax of Docker links.
+	// If not specified, the default is true.
+	// +optional
+	EnableServiceLinks *bool
 }
 
+const (
+	// The default value for enableServiceLinks attribute.
+	DefaultEnableServiceLinks = true
```

Comment:
This should probably exist at the v1 level but not at the "internal" API

Reply:
Removed this.